--- src.orig/Makefile	2003-10-27 18:37:27.000000000 +0000
+++ src/Makefile	2003-10-27 18:39:31.000000000 +0000
@@ -1,8 +1,8 @@
-PREFIX=/usr/local
+PREFIX=/usr
 SHARE_PREFIX=$(PREFIX)/share/xpired
 CFLAGS=`sdl-config --cflags` -DUNIX -DPREFIX=\"$(PREFIX)\" -DSHARE_PREFIX=\"$(SHARE_PREFIX)\"
 LDFLAGS=`sdl-config --libs` -lSDL -lSDL_mixer -lSDL_image -lSDL_gfx
-CC=gcc
+CC=gcc $(CFLAGS) $(OPTFLAGS)
 
 BINARIES = xpired xpiredit
 
@@ -20,11 +20,11 @@
 	rm $(BINARIES) xp*.o
 
 install: xpired xpiredit xpired.lvl xpired.cfg bgimages.txt
-	if [ ! -d $(PREFIX)/bin ]; then mkdir -p $(PREFIX)/bin; fi
-	if [ ! -d $(SHARE_PREFIX) ]; then mkdir -p $(SHARE_PREFIX); fi
-	cp xpired xpiredit $(PREFIX)/bin
-	cp xpired.lvl xpired.cfg bgimages.txt $(SHARE_PREFIX)
-	cp -r img/ $(SHARE_PREFIX)
-	cp -r snd/ $(SHARE_PREFIX)
-	if [ -f readme.txt ]; then cp readme.txt $(SHARE_PREFIX); fi
-	if [ -f xpired.dmo ]; then cp xpired.dmo $(SHARE_PREFIX); fi
+	if [ ! -d $(DESTDIR)$(PREFIX)/bin ]; then mkdir -p $(DESTDIR)$(PREFIX)/bin; fi
+	if [ ! -d $(DESTDIR)$(SHARE_PREFIX) ]; then mkdir -p $(DESTDIR)$(SHARE_PREFIX); fi
+	cp xpired xpiredit $(DESTDIR)$(PREFIX)/bin
+	cp xpired.lvl xpired.cfg bgimages.txt $(DESTDIR)$(SHARE_PREFIX)
+	cp -r img/ $(DESTDIR)$(SHARE_PREFIX)
+	cp -r snd/ $(DESTDIR)$(SHARE_PREFIX)
+	if [ -f readme.txt ]; then cp readme.txt $(DESTDIR)$(SHARE_PREFIX); fi
+	if [ -f xpired.dmo ]; then cp xpired.dmo $(DESTDIR)$(SHARE_PREFIX); fi
